testsuite/features/support/twopence_init.rb

```diff
@@ -220,6 +217,9 @@ def file_inject(node, local_file, remote_file)
 # Other global variables
 $product = product
 $pxeboot_mac = ENV['PXEBOOT_MAC']
+$sle11sp3_terminal = ENV['SLE11SP3_TERMINAL']
+$sle12sp4_terminal = ENV['SLE12SP4_TERMINAL']
+$sle15sp3_terminal = ENV['SLE15SP3_TERMINAL']
```

Comment:
I don't get well why we moved this to gloval variables, but not buildhost ones

Reply:
they are all global variables. The difference is that these are NOT twopence objects, because there's no SSH.